File: src/main/java/cn/xgp/xgplottery/bStats/Metrics.java
package cn.xgp.xgplottery.bStats;

import cn.xgp.xgplottery.XgpLottery;
import cn.xgp.xgplottery.common.FoliaLib.Enum.ServerType;

public class Metrics {
    public static void enable(){
        int pluginId = 18492;
        cn.xgp.xgplottery.bStats.org.bstats.Metrics metrics = new cn.xgp.xgplottery.bStats.org.bstats.Metrics(XgpLottery.instance, pluginId);
        metrics.addCustomChart(new cn.xgp.xgplottery.bStats.org.bstats.Metrics.SimplePie("chart_id", () -> "My value"));
        metrics.addCustomChart(new cn.xgp.xgplottery.bStats.org.bstats.Metrics.SimplePie("isfolia",()-> ServerType.getServerType().name()));
    }
}

File: src/main/kotlin/cn/xgpjun/xgplottery2/bStats/Metrics.kt
package cn.xgpjun.xgplottery2.bStats

import cn.xgpjun.xgplottery2.XgpLottery
import cn.xgpjun.xgplottery2.bStats.org.bstats.Metrics
import cn.xgpjun.xgplottery2.manager.SchedulerManager

object Metrics{
    fun enable() {
        val pluginId = 18492
        val metrics = Metrics(XgpLottery.instance, pluginId)
        metrics.addCustomChart(
            Metrics.SimplePie(
                "chart_id"
            ) { "My value" })
        metrics.addCustomChart(
            Metrics.SimplePie(
                "isfolia"
            ) { SchedulerManager.serverType.name })
    }
}
